src/functionalTest/java/uk/gov/hmcts/reform/wacaseeventhandler/controllers/CaseEventHandlerTestingControllerFunctionalTest.java

```diff
@@ -73,7 +74,7 @@ public void given_post_event_using_test_rest_endpoints_should_create_task() {
             .body("MessageId", equalTo(messageId))
             .body("Sequence", notNullValue())
             .body("CaseId", equalTo(caseIdForTask))
-            .body("EventTimestamp", equalTo(timeStampString))
+            .body("EventTimestamp", equalTo(timeStamp))
```

Comment:
Because you have removed line 64, you will need to change this to match line 177

`timeStamp.toString`